index.html

```diff
@@ -1,14 +1,108 @@
 <!DOCTYPE HTML>
 <html>
-  <head>
-    <title>Mockup Home</title>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
-  </head>
-  <body>
-    <section id="header"></section>
-    <section id="features"></section>
-    <section id="main"></section>
-    <section id="footer"></section>
-  </body>
+
+<head>
+  <title>Mockup Home</title>
+  <link rel="stylesheet" type="text/css" href="style.css">
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
+</head>
+
+<body>
+
+
+
+  <h1>Testing</h1>
+
+  <div class="allPicsDiv">
+    <div class="picDiv"><img src=http://placekitten.com/400/200> Text</div>
+    <div class="picDiv"><img src=http://placekitten.com/400/200> Text</div>
+    <div class="picDiv"><img src=http://placekitten.com/400/200> Text</div>
+  </div>
+
+  <h1>Tell me more</h1>
+  <div class="quoteBanner">
+    This is a great quote

```

Comment:
Again. Wrap your text in text specific tags such as `p` tags or any of the different `h` tags